en/appendices/3-4-migration-guide.rst

```diff
@@ -105,9 +105,6 @@ Other Deprecations
 * ``Router::parse()`` is deprecated. ``Router::parseRequest()`` should be used
   instead as it accepts a request and gives more control/flexibility in handling
   incoming requests.
-* ``Route::parse()`` is deprecated. ``Route::parseRequest()`` should be used
```

Comment:
Oh, not the same base class. The sentences looked similar, I thought it was a duplication. I'll put it back.